src/Chapter1.hs

```diff
@@ -477,8 +476,8 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integral a => a -> a
+lastDigit n = mod v 10
```

Comment:
The implementation in here is almost correct. Unfortunately, it will fail on negative numbers because of how `mod` works for them. Maybe you can think of a way to fix this?

One small note, it won't work correctly on the negative numbers, unfortunately.
Here you would need to use one more standard function – to take the __abs__olute value of n 🙂

Also, it looks like you have a slight typo here
```suggestion
lastDigit n = mod n 10
```

Reply:
I took typo when copying the code from REPL to file... Thank you fro your advice!